extensions/amp-render/1.0/amp-render.js

```diff
@@ -212,7 +225,40 @@ export class AmpRender extends BaseElement {
       },
       'onReady': () => {
         this.toggleLoading(false);
-        this.togglePlaceholder(false);
+        if (this.element.getAttribute('layout') !== Layout.CONTAINER) {
+          this.togglePlaceholder(false);
+          return;
+        }
+
+        let componentHeight, contentHeight;
```

Comment:
Can we add a TODO for using `measureIntersection instead`?

Reply:
Done.